src/main/java/hive/mugshot/storage/ImageUtils.java

```diff
@@ -0,0 +1,65 @@
+package hive.mugshot.storage;
+
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.util.Random;
+import java.util.regex.Pattern;
+
+public final class ImageUtils {
+  private static final String IMAGE_PATTERN = "(^.+\\.(gif|png|bmp|jpeg|jpg)$)";
+
+  private ImageUtils(){
+  }
+
+  public static boolean validateIfHasAnImageAsExtension(final String image){
+    var pattern = Pattern.compile(IMAGE_PATTERN);
+    var matcher = pattern.matcher(image);
+    return matcher.matches();
+  }
+
+  public static BufferedImage resizeImageToSquare(final BufferedImage inputtedImage,final int imageSizeInPixels) {
+    // multi-pass bilinear div 2
+    var bufferedImageWithNewSize = new BufferedImage(imageSizeInPixels, imageSizeInPixels, BufferedImage.TYPE_INT_RGB);
+    var reSizer = bufferedImageWithNewSize.createGraphics();
+    var resizingMode=
+        (inputtedImage.getHeight()<imageSizeInPixels)
+            ? RenderingHints.VALUE_INTERPOLATION_NEAREST_NEIGHBOR
```

Comment:
don't use if ternary
```suggestion
```

Reply:
No problem with ternary operator here.
Spacing needs to be fixed.